Gemfile.lock

```diff
@@ -467,6 +472,8 @@ GEM
       sprockets (>= 3.0.0)
     sqlite3 (1.5.4)
       mini_portile2 (~> 2.8.0)
+    sqlite3 (1.5.4-x86_64-darwin)
+    sqlite3 (1.5.4-x86_64-linux)
```

Comment:
Are these changes worth including, or should I discard them?

Reply:
It is fine.